### PR TITLE
Fixed for current version V

### PR DESCRIPTION
**Pull request by Ddiidev**
Fixed for current version V and added a README

### CODE REVIEW

**Comment by ZihxS** [comment]
@elitru

File: README.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+
+# Jwt
+
+A JWT library for V-Lang
+
+
+## Instalação
```
Comment (ZihxS) [inline README.md]:
```suggestion
## Installation
```

**Review by ZihxS — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+```
+
+```bash
+eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJuYW1lIjoiSm9obiBEb2UiLCJleHAiOjE3MDY3OTg3NDN9.Cj1KkZs5pv1OIGUnqMEBnrnhL_RiN5YCUHmAHHjG0Og
+Test{
+    name: 'John Doe'
+}
+```
+
+## Licença
```
Comment (ZihxS) [inline README.md]:
```suggestion
## License
```

**Review by ZihxS — CHANGES_REQUESTED** [review]

**Comment by Ddiidev** [comment]
@ZihxS Thanks, I lost the readme I made in my native language.

**Comment by ZihxS** [comment]
You're welcome @ldedev

**Comment by ZihxS** [comment]
@elitru

**Review by ZihxS — APPROVED** [review]

**Pull request merged**